src/actions/classifier.js

```diff
@@ -25,6 +25,11 @@ export function startNewClassification(workflowID) {
     }).then(() => {
       return dispatch(setNeedsTutorial())
     }).then(() => {
+      if (getState().classifier.guide[workflowID] !== undefined) {
+        return
+      }
+      return dispatch(fetchFieldGuide(workflowID))
+    }).then(() => {
```

Comment:
I was thinking of putting the returns actually in the call, is that what you're talking about?  Also some of them are dependent on others,  like tutorial is dependent on project preferences.  Or not sure what you're referring to otherwise.